source/reference/minio-mc/mc-ready.rst

```diff
@@ -0,0 +1,106 @@
+============
+``mc ready``
+============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc ready
+
+Syntax
+------
+
+.. start-mc-ready-desc
+
+The :mc:`mc ready` command checks the status of a cluster.
+
+.. end-mc-ready-desc
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following sends a ``GET`` request to the cluster and returns its status.
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc ready play
+
+      The command pings the deployment at the :mc:`~mc alias` ``play``.'
+      The command repeats the request until it is successful.
+      Once the request succeeds, the command outputs the message, ``The cluster is ready``.
+
+      The output resembles the following:
+
+      .. code-block:: shell
+
+         The cluster is unreachable: Get "http://play.min.io:9000/minio/health/cluster": dial tcp 127.0.0.1:9000: connect: connection refused
+         The cluster is ready
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] ready            \
+                          TARGET           \
+                          [--cluster-read] \
+                          [--maintenance]
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: TARGET
+   :required:
+
+   The full path to the :ref:`alias <minio-mc-alias>` or prefix where the command should run.
+
+.. mc-cmd:: --cluster-read
+   :optional:
+
+   Checks if the cluster has enough quorum to serve ``READ`` requests.
```

Comment:
🤔  Is there something useful to link to for `quorum`?

Reply:
Oh there's a glossary entry for `read quorum` that would do. (The ref in there is broken tho.)

Reply:
Added the link and fixed the glossary entry.